src/cli/build.ts

```diff
@@ -120,6 +149,9 @@ export default function (program: RootCmd): void {
             ...cfg.pepr,
             appVersion: cfg.version,
             description: cfg.description,
+            alwaysIgnore: {
+              namespaces: []
+            },
```

Comment:
```suggestion
```

setting `alwaysIgnore.namespaces[]` here would override the config placed in the `pepr` section of `package.json`. This needs to come from cfg.pepr.

Reply:
Fixed in [65bcfc4](https://github.com/defenseunicorns/pepr/pull/1651/commits/65bcfc432198be7227cf87a9b0d3edcccfba5e89)